recipes/pinocchio/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, rmdir, copy
+
+required_conan_version = ">=2"
+
+class PinocchioConan(ConanFile):
+    name = "pinocchio"
+    package_type = "library"
+    license = ("BSD 2-Clause")
+    url = "https://github.com/conan-io/conan-center-index"
```

Comment:
```suggestion
    url = "https://github.com/conan-io/conan-center-index"
    homepage = "https://stack-of-tasks.github.io/pinocchio/"
```

Add project's homepage